cli/internal/fs/turbo_json_test.go

```diff
@@ -134,6 +136,70 @@ func Test_ReadTurboConfig_BothCorrectAndLegacy(t *testing.T) {
 	assert.Equal(t, rootPackageJSON.LegacyTurboConfig == nil, true)
 }
 
+func Test_ReadTurboConfig_InvalidEnvDeclarations1(t *testing.T) {
+	testDir := getTestDir(t, "invalid-env-1")
+
+	packageJSONPath := testDir.Join("package.json")
+	rootPackageJSON, pkgJSONReadErr := ReadPackageJSON(packageJSONPath)
+
+	if pkgJSONReadErr != nil {
+		t.Fatalf("invalid parse: %#v", pkgJSONReadErr)
+	}
+
+	_, turboJSONReadErr := ReadTurboConfig(testDir, rootPackageJSON)
+
+	expectedErrorMsg := fmt.Sprintf("turbo.json: You specified \"$A\" in the \"env\" key. This is treated as a literal, you may want to remove the \"$\"")
+
+	assert.EqualErrorf(t, turboJSONReadErr, expectedErrorMsg, "Error should be: %v, got: %v", expectedErrorMsg, turboJSONReadErr)
+}
+
+func Test_ReadTurboConfig_InvalidEnvDeclarations2(t *testing.T) {
+	testDir := getTestDir(t, "invalid-env-2")
+
+	packageJSONPath := testDir.Join("package.json")
+	rootPackageJSON, pkgJSONReadErr := ReadPackageJSON(packageJSONPath)
+
+	if pkgJSONReadErr != nil {
+		t.Fatalf("invalid parse: %#v", pkgJSONReadErr)
+	}
+
+	_, turboJSONReadErr := ReadTurboConfig(testDir, rootPackageJSON)
+
+	expectedErrorMsg := fmt.Sprintf("turbo.json: You specified \"$A\" in the \"env\" key. This is treated as a literal, you may want to remove the \"$\"")
```

Comment:
The linter warns that this is an unnecessary use of `fmt.Sprintf`, but if I use `fmt.Sprintln`, it adds a `\n` character at the end (and the linter also warns in the corresponding `fmt.Errorf` that errors should not end with a punctuation or new line character. 

<img width="695" alt="CleanShot 2022-09-14 at 21 33 51@2x" src="https://user-images.githubusercontent.com/490968/190315238-060dd3ad-5327-471e-b8b5-c1743f30df79.png">

Reply:
oops, it's late... I had already removed the string interpolation, so this can just be a string literal. Fixed!